spec2nii/GE/ge_pfile.py

```diff
@@ -95,9 +95,9 @@ def _process_svs_pfile(pfile):
         data, meta, dwelltime, fname_suffix = _process_probe_p(pfile)
     elif psd in ('oslaser', 'slaser_cni'):
         data, meta, dwelltime, fname_suffix = _process_oslaser(pfile)
-    elif psd in ('slaser'):
```

Comment:
Note that the above expression would match if `psd` is a **substring** of `'slaser'`. I think that's not what you want:
```python
>>> psd = 'slaser'
>>> psd in ('slaser')
True
>>> psd = 'las'
>>> psd in ('slaser')
True
>>> psd = 'slzzzzz'
>>> psd in ('slaser')
False
>>> 
```

Reply:
Alternatively write:
```python
    elif psd in ('slaser',):
```
Note the comma:
* `('slaser')` is the same as `'slaser'`, a string,
* `('slaser',)` is a tuple containing`'slaser'` as a single element.